_templates/development.html

```diff
@@ -39,23 +39,23 @@ <h2 id="more">{% translate more %}</h2>
 
 <ul class="devprojectlist">
   <li><a href="https://github.com/etotheipi/BitcoinArmory">Armory</a> - A Bitcoin client with enhanced security features.</li>
-  <li><a href="http://bfgminer.com">BFGMiner</a> - Modular Bitcoin mining software.</li>
-  <li><a href="https://code.google.com/p/bitcoinj/">bitcoinj</a> - A Java implementation of a Bitcoin client-only node used in thin SPV Bitcoin clients.</li>
-  <li><a href="https://github.com/schildbach/bitcoin-wallet/">Bitcoin Wallet for Android</a> - A thin SPV Bitcoin client for mobiles.</li>
-  <li><a href="https://github.com/bitsofproof/supernode">Bits of Proof Enterprise Bitcoin Server</a> - A modular implementation of the Bitcoin protocol in Java.</li>
+  <li><a href="http://bfgminer.com">BFGMiner</a> - A modular Bitcoin miner.</li>
+  <li><a href="https://en.bitcoin.it/wiki/Bitcoin_Explorer">Bitcoin Explorer</a> - A Bitcoin command line tool, built on libbitcoin.</li>
+  <li><a href="https://en.bitcoin.it/wiki/Bitcoin_Server">Bitcoin Server</a> - A Bitcoin full node and query server, built on libbitcoin.</li>
+  <li><a href="https://github.com/schildbach/bitcoin-wallet">Bitcoin Wallet</a> - A thin SPV Bitcoin client for Android.</li>
+  <li><a href="https://bitcoinj.github.io">bitcoinj</a> - A Java Bitcoin client library for SPV clients.</li>
   <li><a href="https://github.com/btcsuite/btcd">btcd</a> - A full node bitcoin implementation written in Go.</li>
   <li><a href="https://github.com/btcsuite/btcwallet">btcwallet</a> - A hierarchical deterministic wallet daemon written in Go.</li>
-  <li><a href="https://electrum.org/community.html">Electrum</a> - A fast Bitcoin client relying on remote servers to store the block chain.</li>
+  <li><a href="https://electrum.org">Electrum</a> - A fast Bitcoin client relying on remote servers to store the blockchain.</li>
   <li><a href="https://github.com/luke-jr/eloipool">Eloipool</a> - A fast Python mining pool server software.</li>
-  <li><a href="https://github.com/hivewallet">Hive</a> - A fast user-friendly SPV Bitcoin client.</li>
-  <li><a href="https://github.com/libbitcoin/libbitcoin/">libbitcoin</a> - An asynchronous C++ library for Bitcoin.</li>
-  <li><a href="https://en.bitcoin.it/wiki/Libblkmaker#For_developers">libblkmaker and python-blkmaker</a> - Client side libraries for the getblocktemplate mining protocol.</li>
-  <li><a href="https://multibit.org/">MultiBit HD</a> - A thin SPV international Bitcoin client for desktops.</li>
+  <li><a href="https://github.com/hivewallet">Hive</a> - A fast user-friendly SPV Bitcoin wallet.</li>
+  <li><a href="https://en.bitcoin.it/wiki/Libbitcoin">Libbitcoin</a> - A Bitcoin cross-platform C++ development toolkit.</li>
+  <li><a href="https://gitlab.com/bitcoin/libblkmaker">Libblkmaker</a> - A C client library for the getblocktemplate mining protocol.</li>
+  <li><a href="https://multibit.org">MultiBit HD</a> - A thin SPV international Bitcoin client for desktops.</li>
```

Comment:
Yes, it should really be more consistent. The term client is confusing as used. I've made an additional PR to improve consistency between wallet and client.